src/libsyntax/ast.rs

```diff
@@ -782,6 +865,9 @@ impl BinOpKind {
 
 pub type BinOp = Spanned<BinOpKind>;
 
+/// Unary operator.
+///
+/// Plase note that `&data` is not an operator, it's an `AddrOf` expression.
```

Comment:
```suggestion
/// Note that `&data` is not an operator, it's an `AddrOf` expression.
```
(Has a typo and looks a bit strange in this context.)